vmm/src/acpi.rs

```diff
@@ -264,6 +264,18 @@ fn create_mcfg_table(pci_segments: &[PciSegment]) -> Sdt {
     mcfg
 }
 
+fn create_tpm2_table() -> Sdt {
+    let mut tpm = Sdt::new(*b"TPM2", 52, 3, *b"CLOUDH", *b"CHTPM2  ", 1);
+
+    tpm.write(36, 0_u16); //Platform Class
+    tpm.write(38, 0_u16); // Reserved Space
+    tpm.write(40, 0xfed4_0040_u64); // Address of Control Area
```

Comment:
Can't we generate the address from our MMIO allocator so that we're not hardcoding all over the codebase?